### DISCUSSION TITLE
How do I delete a script?

### DISCUSSION THREAD

**Discussion by 3200pro** [johnlindquist/kit#657] [Q&A]
Guessing this is pretty obvious however I'm missing where to do this. How do I remove a script from the 'run' panel of Script Kit?

**Comment by johnlindquist** [comment]
1. Pressing the "right" in your keyboard with any script selected brings up the actions you can take. Delete is one of them.

2. cmd+shift+backspace with a script selected will prompt to delete it

3. Adding //Exclude: true to any script will hide it (without deleting) from the main menu. This allow you to still use keyboard shortcuts, etc

4. Kit->Manage Kevs->Remove->examples should clear out the examples scripts.

5. Delete a script from your editor of choice and the app will notice it's gone.

**Reply by johnlindquist** [reply]
I'll also share how you can build your own "main menu" scripts if you prefer more customization.

await selectScript()

Will bring up a clean script menu.

**Reply by 3200pro** [reply]
Thanks for the quick info. For some reason on my mac right click didn't do anything. (tried it initially)

the cmd+shift+delete brought up the delete script [Y] [N] prompt. I am now free of Corgi's. Thanks:)

**Reply by johnlindquist** [reply]
"right" as in "right arrow" on the keyboard. Then "left" would take you back. It's a very keyboard-centric tool.

**Reply by 3200pro** [reply]
Thanks! That's an interesting method I've never encountered. Right arrow brings up a ton of options. Very cool feature. Right arrow may be just abstract enough to need some indication that it's an option. Let me know if that's already incorporated and I missed it.
